### ISSUE TITLE
Document migration path for upgrade from `helm template` to `helm upgrade`

### ISSUE THREAD

**Issue by axeal** [rancher/rancher-docs#1095]
## Summary

Document a migration path for existing Rancher users who initially installed Rancher via `helm template` to permit upgrade via `helm upgrade`, as is default for Rancher v2.6+

## Details

Via https://github.com/rancher/rancher-docs/issues/982#issuecomment-1850612486 and https://github.com/rancher/rancher-docs/pull/1066 the Rancher v2.6+ documentation now includes a section on the upgrade of Rancher via `helm template` for users who installed their Rancher instance via `helm template`, which was the documented method for air-gap installation in Rancher v2.x < v2.6.

The request here is to document the required steps for a user to migrate from using `helm template` to `helm upgrade` for the upgrade, such that there is a single consistent air-gap upgrade path going forward.

One way this would be possible is by performing a `helm upgrade --install` repeatedly, and adding the missing labels/annotations to each resource as noted by the helm output, e.g.:

```
$ helm upgrade --install -n cattle-system rancher rancher-stable/rancher --version=2.7.9 --set hostname=rancher.example.com
Release "rancher" does not exist. Installing it now.
Error: Unable to continue with install: PriorityClass "rancher-critical" in namespace "" exists and cannot be imported into the current release: invalid ownership metadata; label validation error: missing key "app.kubernetes.io/managed-by": must be set to "Helm"; annotation validation error: missing key "meta.helm.sh/release-name": must be set to "rancher"; annotation validation error: missing key "meta.helm.sh/release-namespace": must be set to "cattle-system"
```

```
kubectl label priorityclass rancher-critical app.kubernetes.io/managed-by=Helm
kubectl annotate priorityclass rancher-critical meta.helm.sh/release-name=rancher
kubectl annotate priorityclass rancher-critical meta.helm.sh/release-namespace=cattle-system
```

... and continuing for each reported resource with missing labels/annotations until the upgrade is successful

**Comment by martyav** [comment]
@axeal Are the missing labels and annotations common enough from user to user that we could include a list?

**Comment by axeal** [comment]
@martyav Yes, that set of label and annotations will be the same for every resource created by the rancher helm chart (assuming it was installed with the release name rancher and into the cattle-system namespace, per the installation documention). The set of resources created by the chart may vary depending upon helm chart values, and the `helm upgrade --install` command will only return the error for the first resource upon which it fails and exits, so the user would need to run the `helm upgrade --install` multiple times, adding the label and annotations to each resource for which the error is reported, until the command runs without error.
